src/util/get-project-root.js

```diff
@@ -2,4 +2,4 @@ import partial from 'lodash/partial';
 import getCmdStdout from './get-cmd-stdout';
 const cmd = 'git rev-parse --show-toplevel';
 
-export default partial(getCmdStdout, cmd);
+export default (cwd) => partial(getCmdStdout, cmd, { cwd })();
```

Comment:
(also note removed parentheses around `cwd` to match style—AirBnb ESLint doesn’t care but the rest of the code omits them)